packages/cli/src/commands/configure/configure_profile_command.test.ts

```diff
@@ -45,6 +53,7 @@ void describe('configure command', () => {
       mockPrint.mock.calls[0].arguments[0] as string,
       /already exists!/
     );
+    assert.equal(emitSuccessSpy.mock.callCount(), 1);
```

Comment:
add the arguments assertion as well, i.e. what was the command and in failures, what was the error field of `usageData` sent